### PR TITLE
Vagrant box for dev environment

### PR DESCRIPTION
**Pull request by d-rivera-c**
## Description
To help get a dev env running more easily I've created a vagrant box for myself and thought it might be useful for someone else. It has the basic dependencies installed and instructions to install everything else once the box is running. Runs Ubuntu Bionic and installs ruby + postgres on the first set up, and there are instructions in the Vagrantfile to install everything else and run the server from the box.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation change

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,18 +23,27 @@ https://github.com/WhareHauora/wharehauora-server/wiki
 Installation
 ============
 
+## Prerequisites
+
+You need to have these set up in your machine:
+* Ruby 2.4.1
+* Postgres
+
+Or there's now a [vagrant box](https://www.vagrantup.com/intro/getting-started/) available with Ruby + Postgres installed already to make it easy to set up a dev environment. You'll be able to access the server in http://127.0.0.1:3000/. Read the [Vagrantfile](Vagrantfile.md) comments for installation help.
```
Comment (Br3nda) [inline README.md]: Can we rephrase to something line

```
There are two ways to set up a dev environment: running in ruby and bundler or using a pre-build Vagrant box

## Ruby + Bunder
... intructions

## Vagrant
... instructions
```

Comment (d-rivera-c) [reply README.md]: Ah yes, thank you for pointing that out. I'll try to make things clearer :)

**Comment by d-rivera-c** [comment]
I've updated most of the linting errors noted in `README` but some don't make sense enforcing (like line length of the AwesomeCode badge line).
I'm done making the changes so this can be peer reviewed again :)

**Comment by Br3nda** [comment]
Thank you for this.

**Pull request merged**